### PR TITLE
Add basic search in

### PR DESCRIPTION
**Pull request by andrewfluck**
I changed the name of the branch because I was an idiot, and named it `feature/develop`, All the commits are just a side effect it seems

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -132,4 +149,27 @@ const getAudienceReviews = async (slug, reviewCount, isTV = false) => {
     )
 }
 
-module.exports = { getAudienceReviews }
+const getMovieResults = async (query) => {
```
Comment (grikomsn) [inline index.js]: Adding a comment block on what `getMovieResults` does would be great, including what parameters are needed, what does it return and/or throw, and so on.

File: index.js [hunk 2]
```diff
@@ -64,6 +79,8 @@ const scrapePage = data => {
  * @param {Number} reviewCount Number of requested reviews to be given
  * @param {Boolean} isTV State whether given slug is a TV show or not
  */
+
```
Comment (grikomsn) [inline index.js]: Any reason why is there two line breaks?

Comment (andrewfluck) [reply index.js]: Probably accidental, I'll fix that

**Comment by andrewfluck** [comment]
Was there more linebreaks?

**Comment by andrewfluck** [comment]
Is there any reason we can't merge? I have made the changes I think?

**Comment by grikomsn** [comment]
While @vimark1 have approved, `index.js` has some conflicts that must be resolved.

**Comment by vimark1** [comment]
> While @vimark1 have approved, `index.js` has some conflicts that must be resolved.

Conflict has been resolved

**Pull request merged**